Modified css and templates to match theme

Merlingilb opened the pull request.

Merlingilb committed:
Merge pull request #1 from Jujuedv/master

pull

Commit:
-modified css and templates to match the theme

Commit:
-modified holomail images to a more pleasing look.

Commit:
-modified css with a more "modern" layout and with improved switch pages in holomail

TurnTheTideTM commented:
![image](https://cloud.githubusercontent.com/assets/5428121/23863845/1f115910-0811-11e7-8e7d-c1fb0a85be34.png)

Die Holomail-Liste ist nicht zentriert/nicht volle Breite, kannst du das noch fixen?

Außerdem scheint das Holomail-Bild den rest der Seite ein paar pixel nach unten zu verschieben - die Menüleiste ist auf der Login-Seite 34px hoch wenn nicht eingeloggt, und 34.95px wenn eingeloggt.

Merlingilb committed:
-changed logo to a more fitting and pleasing design
-changed "jetzt mitspielen" picture to another design

Merlingilb commented:
Ich wollte heute sowieso mit m Juli reden, ob wir das nicht sowieso komplett anders machen wollen.
Dann kann man sich auf so etwas noch eher konzentrieren.

Aber komisch ist es schon weil ich eigentlich alles getestet hatte.

Merlingilb committed:
-fixed holomail table alignment

Merlingilb committed:
-changed richards complaints

TurnTheTideTM committed:
Fix url for next/prev page

Next/Previous buttons now redirect to other pages of the outbox, not the inbox.
